samples/zigbee/ncp/src/main.c

```diff
@@ -53,19 +47,13 @@ zb_ret_t zb_osif_bootloader_run_after_reboot(void)
 {
 #if DT_NODE_EXISTS(DT_ALIAS(rst0))
 	int err = 0;
-	const struct device *rst_dev = device_get_binding(RST_PIN_PORT);
+	const struct gpio_dt_spec rst0 = GPIO_DT_SPEC_GET(DT_ALIAS(rst0), gpios);
 
-	if (!rst_dev) {
-		return RET_ERROR;
-	}
-
-	err = gpio_pin_configure(rst_dev, RST_PIN_NUMBER, GPIO_OUTPUT);
-	if (err) {
+	if (!device_is_ready(rst0.port)) {
 		return RET_ERROR;
 	}
 
-	/* Perform pin reset */
-	err = gpio_pin_set_raw(rst_dev, RST_PIN_NUMBER, RST_PIN_LVL);
+	err = gpio_pin_configure_dt(&rst0, GPIO_OUTPUT_ACTIVE);
```

Comment:
This does not seem to be correct. The idea was to set the reset pin to low, so the nRF5 BL will boot, detect the GPIO pin-based reset and start the DFU over USB.